src/Models/BaseModel.php

```diff
@@ -79,6 +79,20 @@ public function saveWithoutTouch()
         $this->timestamps = true;
     }
 
+    public function deleteWithoutTouch()
```

Comment:
Feels like all these `...WithoutTouch()` methods could call a helper method that gets passed the operation function to be called.